tests/unit/api/search/test_v0.py

```diff
@@ -286,6 +287,7 @@ def test_search_user_success(self) -> None:
         Test request success
         :return:
         """
+        self.maxDiff = None
```

Comment:
do we use this?

Reply:
Nope. this was only to debug the test case. Will remove this.